cli/mender-artifact/read.go

```diff
@@ -57,7 +57,7 @@ func readArtifact(c *cli.Context) error {
 	ver := func(message, sig []byte) error {
 		sigInfo = "signed but no key for verification provided; " +
 			"please use `-k` option for providing verification key"
-		if verifyCallback != nil {
+		if c.String("key") != "" {
```

Comment:
Suggestion: sigInfo is assigned to above, but the original string is never used. Remove? Also, the sentence: `signed; verification using provided key failed` should maybe drip the semicolon?

Reply:
AFAICS all of them are used. Which one are you referring to?

Reply:
I was originally thinking the `no-signature` but It's used below (which makes sense), sry.